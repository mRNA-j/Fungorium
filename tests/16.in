create_mushroom_picker Bob mp1
create_tecton normal t1
create_tecton normal t2
add_neighbour_to_tecton t1 t2
create_mushroom Bob t1 m1
create_yarn m1 normal y1
action_grow_yarn t1 t2 y1
create_spore_on_tecton t2 accelerator
create_spore_on_tecton t2 accelerator
create_spore_on_tecton t2 accelerator
action_grow_mushroom t2 mp1 m2
stat