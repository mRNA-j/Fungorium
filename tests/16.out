Tecton: t1
Type: normal
Mushroom: m1
Insects: none
Yarns: y1
Neighbours: t2
Spores: none

Tecton: t2
Type: normal
Mushroom: m2
Insects: none
Yarns: y1
Neighbours: t1
Spores: none

Mushroom: m1
HasSpore: false
Place: t1
Owner: Bob

Mushroom: m2
HasSpore: false
Place: t2
Owner: Bob

Yarn: y1
Type: normal
Tectons in yarn: t1 t2

Player: Bob
Type: MushroomPicker
Points: 3

